tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml

```diff
@@ -45,10 +70,16 @@ jobs:
           --volume $(Build.SourcesDirectory):/onnxruntime_src \
           --volume $(Build.BinariesDirectory):/build \
           --volume $HOME/.onnx:/home/onnxruntimedev/.onnx \
+          --volume $(Pipeline.Workspace)/ccache:/cache \
           -e ALLOW_RELEASED_ONNX_OPSET_ONLY=0 \
           -e NIGHTLY_BUILD \
           -e BUILD_BUILDNUMBER \
+          -e CCACHE_DIR=/cache \
           onnxruntimecpubuild \
+          /bin/bash -c "
+            set -ex; \
+            env; \
```

Comment:
Be careful on this. Sometime envs may contain sensitive data, for example, SAS keys.

Reply:
Removed